base/system/smss/smss.c

```diff
@@ -70,7 +70,7 @@ SmpExecuteImage(IN PUNICODE_STRING FileName,
     if (!NT_SUCCESS(Status))
     {
         /* This is a pretty bad failure. ASSERT on checked builds and exit */
-        ASSERTMSG("RtlCreateProcessParameters", NT_SUCCESS(Status));
+        ASSERTMSG("RtlCreateProcessParameters failed :: ", NT_SUCCESS(Status));
```

Comment:
`failed: ` (one colon, no space before, one space after).
Or is it meant to be a simple separator "symbol" ?

Reply:
Yeah it seems it's supposed to be a separator. Just using \n like everywhere else seems preferable.

Reply:
Yes, an arbitrary separator "symbol".
My idea was "everything on one line, so the actual assertion is not missed, except `FALSE` which we don't care about",
so I would rather suggest to use ` :: ` everywhere.
What do you choose?

Reply:
Done.